tests/test_deeppath.py

```diff
@@ -126,17 +120,17 @@ def test_dget_flatten_and_repetition():
     assert dget(data1, "*") == [reps]
     assert dget(data1, "flattened[0]") == {"nested_in_rep": 1}
     # */ is a list, it needs explicit unfold
-    assert dget(data1, "*/nested_in_rep") is None
+    assert dget(data1, "*/nested_in_rep") == []
     assert dget(data1, "*[*]/nested_in_rep") == [1, 2]
```

Comment:
I think there were some edge cases where this kind of behaviour didn't work, please check in Jira issues I did report it. I guess it's different now though since we now have wildcard support.

Reply:
Please do check it may or may not have been closed by me in Jira but I think we should be wary if it's going to cause existing versions to break just search for `dget` in the Jira project that tracked issues for this.

Reply:
It looked correct, I was just worried things may break so I wanted to sanity check it. I can't remember if I had added an xfail test for this, I don't think I did... Ah well.